tasks/21_check-letters/check-letters.js

```diff
@@ -0,0 +1,8 @@
+export function checkLetters(arr) {
+  const lower = arr[0].toLowerCase();
+  const searchLower = arr[1].toLowerCase();
+  for (let i=0; i< searchLower.length; i++) {
```

Comment:
I'm the last person to correct you naming vars since I'm very bad at it, but please try naming it more meaningful like firstString, secondString. Also please consider writing this `for (let i=0; i< searchLower.length; i++) {` with spaces like this `for (let i = 0; i < searchLower.length; i++) {`

Reply:
FIxed, and I will try to be more precise with naming variables in the future.